dailydotdev/apps
# fix: add a close button to comment input

## Changes

This PR resolves #1606 https://github.com/dailydotdev/daily/issues/1606



- Added a CloseButton component to the MarkdownInput component, so that we can see a "X" on the top-right corner of the comment box.
<img width="546" alt="image" src="https://github.com/user-attachments/assets/caad2a12-9c5f-40af-86c8-8c61112a3d6d">

- Let CommentInputOrModal passes onClose to CommentMarkdownInput as a prop "onRequestClose". This function is eventually passed to MarkdownInput, so that clicking the close button can cancel the comment.

## Manual Testing

> [!CAUTION]
> Please make sure existing components are not breaking/affected by this PR
1. Go to a post
2. Click on comment button
3. Should see the comment box with a close button. Clicking the close button will cancel the comment.

-->
